### ISSUE TITLE
Question about pre-trained weight loading

### ISSUE THREAD

**Issue by endaoguansanlu** [open-mmlab/mmsegmentation#2141]
Hello, can mmseg load two pre-training weights, for example, a two-branch model, one branch is segformer, one branch is resnet, I will load the two pre-training weights to this two-branch model and then perform it on cityscapes segmentation.

**Comment by MengzhangLI** [comment]
No. But you can defined a new backbone model which includes SegFormer backbone and ResNet , then save these two pretrained models(backbone) into one. Finally you can load this single one pretrained model.

Best,
